gelf/message.go

```diff
@@ -71,7 +71,7 @@ func NewMessage(l Level, short string, full string) *Message {
 		Host:         host,
 		ShortMessage: short,
 		FullMessage:  full,
-		Timestamp:    time.Now().UnixNano(),
+		Timestamp:    float64(time.Now().UnixNano()) / 1e9,
```

Comment:
that gives you only the seconds. I thought it would be nice to keep the fractions of seconds, which is valid GELF